src/element-style-observer.js

```diff
@@ -234,10 +223,12 @@ export default class ElementStyleObserver {
 
 		properties = [...new Set(properties)]; // Dedupe
 
-		// Only add properties not already present and not excluded
+		const allowDiscrete = CSS.supports("transition-behavior", "allow-discrete") ? " allow-discrete" : "";
```

Comment:
I meant you can set it **outside the class**. This is not going to change the entire time the page is loaded (it could only change if the browser is updated!)

Reply:
Fixed!